pages/common/st-util.md

```diff
@@ -0,0 +1,15 @@
+# st-util
+
+> Run GDB server to interact with STM32 ARM Cortex microcontoller.
+
+- Run GDB server on port 4500:
+
+`st-util -p 4500`
```

Comment:
Sure that makes sense. Its there in CONTRIBUTING.md https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines. But yea, its not mandatory.